cmd/device/adminstate/update/update.go

```diff
@@ -0,0 +1,68 @@
+package update
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"github.com/edgexfoundry/edgex-cli/config"
+	"github.com/spf13/cobra"
+	"strings"
+
+	"github.com/edgexfoundry/go-mod-core-contracts/clients"
+	"github.com/edgexfoundry/go-mod-core-contracts/clients/metadata"
+	"github.com/edgexfoundry/go-mod-core-contracts/clients/urlclient/local"
+	"github.com/edgexfoundry/go-mod-core-contracts/models"
+	"github.com/edgexfoundry/go-mod-core-contracts/requests/states/admin"
+)
+
+var deviceName string
+var deviceId string
+var state string
+
+// NewCommand returns the update deviceName command
+func NewCommand() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "update",
+		Short: "Update deviceName admin state",
+		Long:  `Update deviceName admin state`,
+		RunE:  updateAdminStatusHandler,
+	}
+	cmd.Flags().StringVarP(&state, "state", "s", "", fmt.Sprintf("Admin State \nValues: [%s]", strings.Join([]string{models.Locked, models.Unlocked}, ",")))
+	cmd.Flags().StringVarP(&deviceName, "deviceName", "d", "", "Name of the deviceName to be updated")
```

Comment:
Would it have been just as clear to have flags such as 
n "name".   and
i "id"

Reply:
Agree!

Reply:
Resolved